pkg/oc/cli/importimage/importimage.go

```diff
@@ -481,23 +526,33 @@ func (o *ImportImageOptions) newImageStreamImportAll(stream *imageapi.ImageStrea
 	return isi
 }
 
-func (o *ImportImageOptions) newImageStreamImportTags(stream *imageapi.ImageStream, tags map[string]string) *imageapi.ImageStreamImport {
+func (o *ImportImageOptions) newImageStreamImportTags(stream *imageapiv1.ImageStream, tags map[string]string) *imageapiv1.ImageStreamImport {
 	isi, streamInsecure := o.newImageStreamImport(stream)
 	for tag, from := range tags {
 		insecure := streamInsecure
 		scheduled := o.Scheduled
-		oldTag, ok := stream.Spec.Tags[tag]
```

Comment:
cc @bparees 
External image api does not allow maps, and instead stores tags as a slice. Is there any preference here as to how tag lookups should occur?

Reply:
nope, iterating seems reasonable to me.